Marlin/src/lcd/menu/menu_main.cpp

```diff
@@ -101,14 +100,15 @@ void menu_main() {
 
   if (busy) {
     MENU_ITEM(function, MSG_PAUSE_PRINT, lcd_pause);
-    MENU_ITEM(submenu, MSG_TUNE, menu_tune);
-  }
-  else {
-    MENU_ITEM(function, MSG_RESUME_PRINT, lcd_resume);
     #if ENABLED(SDSUPPORT)
       if (card.isFileOpen())
         MENU_ITEM(submenu, MSG_STOP_PRINT, menu_sdcard_abort_confirm);
     #endif
+    MENU_ITEM(submenu, MSG_TUNE, menu_tune);
+  }
+  else {
+    MENU_ITEM(function, MSG_RESUME_PRINT, lcd_resume);
```

Comment:
Its set to send a resume to remote streamers, such as octoprint, in cases where they fail to resume automatically. If it does not have a file open from SD, it will just send the resume action command.

Reply:
@InsanityAutomation 
hm, but I think it's not good. Maybe you should think about a config setting, if necessary.

Reply:
I could add some extra conditionals so that it will only display if a resume action is defined, or if card is open. If paused from the streamer, the firmware simply has no way to know if its been paused or canceled. In my case, octoprint routinely misses the resume command after pausing, im at the LCD to change filament and this allows me to just click that to resume the print without needing to get out a web interface. Another reason it makes sense is the LCD pause button can now send the pause action command if configured, so a print can be paused and restarted solely from the LCD back to octoprint or repetiere.

Granted I believe the root of this at filament change at least is that when pausing, the serial buffer fails to send the last OK until after its resumed, but thats something to dig into another day.

Reply:
Then `ADVANCED_PAUSE_FEATURE && IS_SD_PRINTING` should be a condition - or something like that?

Reply:
Okay, I did not look at the code.

Reply:
Did this get resolved? I think this PR was merged a little too quickly. I should've marked it as "Needs Work" to encourage more testing and refinement.

Reply:
Yes, this was merely a debate on positioning between button or dial, and we made it configurable in a different pr.